SSA/Projects/InstCombine/ForLean.lean

```diff
@@ -299,22 +298,18 @@ def sdiv_one_one' (h : 1 < w) : BitVec.sdiv 1#w 1#w = 1#w := by
   apply BitVec.eq_of_toNat_eq
   simp [hone]
 
-def sdiv_one_one : BitVec.sdiv (BitVec.ofInt w 1) 1 = 1 := by
+def sdiv_one_one : BitVec.sdiv 1#w 1 = 1 := by
```

Comment:
This is not in simp-normal-form for sure, right? It's depending on the `1 : Nat` to get coerced into `1#w : BitVec w`? A good way to check that the theorem statement is in simp normal form is to write 

```suggestion
def sdiv_one_one : BitVec.sdiv 1#w 1 = 1 := by simp
```

and see if `simp` complains about the theorem statement being in normal form. I would imagine that the canonical theorem statement is

```lean
def sdiv_one_one : BitVec.sdiv 1#w 1#w = 1#w := by simp
```

Reply:
Resolved.